specification/source/cxf.rst

```diff
@@ -0,0 +1,290 @@
+  sequence from CDL to CXF. 
+
+* For Elementary Blocks (:numref:`sec_ele_blo`), their
+  CXF representation does not include the implementation 
+  (``equation`` section).
+
+* Like many scientific modeling languages, Modelica requires 
+  tight casting of data types. Commercial control products are 
+  more flexible in the use of data types. 
+  
+  [For example, in Modelica, a data type needs to be declared as
+  type ``Real`` or ``Integer``. ``Real`` data are not allowed to be 
+  tested for equality since computations are prone to rounding 
+  errors. Many commercial control systems define continuous 
+  data types as ``analog``. 
+  ]
+
+* Libraries which utilize matrices and vectors in CDL shall have
+  the option to be modified (or "flattened") to result in an 
+  instance of control logic which is then represented in CXF. 
+  [For example: A[1] and B[1,2] are elements of a matrix. Then, 
+  after flattening, their names are A_1 and B_1_2. A similar 
+  pattern applies for higher dimensional matrices.  
+  ]
+
+* Certain Blocks that are supported in CDL are not readily 
+  supported in commercial control products. 
+  [Some examples include:
+   
+  * Blocks that utilize vectors and matrices. These blocks are 
+    used for the development and support of control sequence 
+    libraries but are not easily utilized in graphical 
+    programming tools. 
+  * The “Pre” block which is used to break algebraic loops that
+    involve Boolean variables during a simulation but is not 
+    required in common building controllers.
+
+  ]
+
+Classes and Properties
+^^^^^^^^^^^^^^^^^^^^^^
+
+A valid CXF file contains Blocks (that is Elementary Blocks, 
+Composite Blocks, Extension Blocks or a combination of these) and
+each instance of a block uses the set of input and output 
+connectors, parameters, and constants as defined within definition 
+of the Block. To support this process, a Resource Description 
+Framework (RDF) graph representation of the standard has been 
+provided in a CXF-Core.jsonld file using the MIME type 
+``application/ld+json`` and it can be found `here <https://github.com/lbl-srg/modelica-json/blob/master/CXF-Core.jsonld>`_.
+The key classes and properties present in CXF-Core.jsonld 
+that can be used to created CXF classes are shown in
+Table :numref:`tab_cxf_cla` and Table :numref:`tab_cxf_rel` respectively. 
+
+
+.. _tab_cxf_cla:
+
+.. table:: Key classes within ``CXF-Core.jsonld``
+   :widths: 15 80
+
+   ============================  ===========================================================
+   Class                         Description
+   ============================  ===========================================================
+   Package						 
+   								 A package is a specialized class used to group multiple 
+   								 blocks.
+   Blocks   					 A block is the abstract interface of a control sequence.
+   ElementaryBlock 
+   								 An elementary block defined by ASHRAE S231P (subClassOf 
```

Comment:
Above you use capital letters for Elementary Block, Composite Block etc which is good, but here in this table you use lower case. Change to capital letters to be consistent.